bootstrap/kubeadm/controllers/suite_test.go

```diff
@@ -53,8 +53,8 @@ var _ = BeforeSuite(func() {
 		defer GinkgoRecover()
 		Expect(testEnv.StartManager(ctx)).To(Succeed())
 	}()
-
 	<-testEnv.Manager.Elected()
+	testEnv.WaitForWebhooks()
```

Comment:
What about updating our testing guidelines here https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/testing.md?

Reply:
Already part of the changes.

Reply:
missed that 😓